src/status_im/transport/db.cljs

```diff
@@ -35,3 +37,41 @@
    :fetch-history?        true
    :resend?               resend?
    :topic                 topic})
+
+(spec/def ::profile-image :contact/photo-path)
+
+(spec/def :chat/name (spec/nilable string?))
+
+(spec/def :group-chat/admin :global/public-key)
+(spec/def :group-chat/participants (spec/coll-of :global/public-key :kind set?))
+(spec/def :group-chat/signature string?)
+
+(spec/def ::content (spec/conformer (fn [content]
+                                      (cond
+                                        (string? content) {:text content}
+                                        (map? content) content
+                                        :else :clojure.spec/invalid))))
```

Comment:
We should probably validate map content more throughly, dependent on whether it's `command` or `text/plain`/`emoji` content type, certain keys has to be there in order to not crash the rendering. 
For `text/plain` it's just `:text` which has to be string (can by simply conformed to string by calling `(update content :text str)`, for command, it's little bit more complicated and I will provide you with map of required keys/types-of-keys.

Reply:
So the spec:

`content-type`:`text/plain`
It has to be a map with required key `:text` of type string, optional key `:response-to` of type string as well.

`content-type`:`command`
It has to be a map with required keys `:command-path` of type `[string set(keywords or strings)]`  and `:params` of type `map(keyword->anything)`

Reply:
done